packages/odyssey-react/src/components/FieldLabel/FieldLabel.stories.tsx

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { Story } from "@storybook/react";
+import React from "react";
+import FieldLabel from ".";
+import type { Props } from ".";
+
+export default {
+  title: `Components/FieldLabel`,
```

Comment:
Out of curiosity - is the reason we're wrapping all these `title` fields in back-ticks due to a lint issue?

Reply:
no - just copy-pasta from other files to get the prototype components up and running. Your feedback about writing code in a single voice is a good one and I have plans to introduce more formalized linting :soon: Changing these should be easy via a `--fix` flag so I haven't been too concerned just yet!